cpp/src/parquet/arrow/reader.h

```diff
@@ -145,22 +145,6 @@ class PARQUET_EXPORT FileReader {
   virtual ::arrow::Status ReadColumn(int i,
                                      std::shared_ptr<::arrow::ChunkedArray>* out) = 0;
 
-  // NOTE: Experimental API
-  // Reads a specific top level schema field into an Array
-  // The index i refers the index of the top level schema field, which may
-  // be nested or flat - e.g.
-  //
-  // 0 foo.bar
-  //   foo.bar.baz
-  //   foo.qux
-  // 1 foo2
-  // 2 foo3
-  //
-  // i=0 will read the entire foo struct, i=1 the foo2 primitive column etc
```

Comment:
The docstring might be good to keep as part of ReadColumn? Or does it mean something different?

Reply:
Double-checked and yeah, that comment still applies. I'll add it back.